bugsnag-plugin-android-anr/src/main/jni/bugsnag_anr.c

```diff
@@ -0,0 +1,18 @@
+#include "bugsnag_anr.h"
+#include "anr_handler.h"
+
+#include <jni.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+JNIEXPORT void JNICALL Java_com_bugsnag_android_AnrPlugin_installAnrDetection(
+    JNIEnv *env, jobject _this, jobject byteBuffer) {
+    bsg_handler_install_anr((*env)->GetDirectBufferAddress(env, byteBuffer));
```

Comment:
There should be a null check for the buffer, just for sanity. If for whatever reason it could not be created, it would be an instant crash.